templates.go

```diff
@@ -12,7 +13,12 @@ func loadTemplates(dir string) *template.Template {
 		return getTemplates()
 	}
 	logger.Printf("using custom template directory %q", dir)
-	t, err := template.New("").ParseFiles(path.Join(dir, "sign_in.html"), path.Join(dir, "error.html"))
+	funcMap := template.FuncMap{
+		"ToUpper": strings.ToUpper,
+		"ToLower": strings.ToLower,
+	}
+	logger.Printf("using custom template directory %q", dir)
```

Comment:
This log line is duplicated with line 15

Reply:
I’ll take a stab at a test. Not sure how I missed that duplicate logging entry...

Reply:
Tests added... PTAL

Reply:
CHANGELOG entry added

Reply:
sorry about the lint error.. I used gitpod and my editor settings weren't configured there to format on save